layerserver/utils.py

```
@@ -102,18 +101,25 @@ def geojsonlayer_remote_data(url, headers):
     return result
 
 
+def geojsonlayer_get_fields(data):
+    fields = []
+    try:
+        if 'Features' in data and len(data['Features']) > 0:
+            fields = list(data['Features'][0]['Properties'].keys())
+        if 'features' in data and len(data['features']) > 0:
+            fields = list(data['features'][0]['properties'].keys())
+    except Exception:
```

@abusquets silent error